projects/cashmere/src/lib/tabs/tab-set/tab-set.component.ts

```diff
@@ -187,7 +188,7 @@ export class TabSetComponent implements AfterContentInit {
         this.setUpTabs(false);
         this.refreshTabWidths();
 
-        // If using routing and routerLinkActive hasn't set any active tabs, default to the first route
+        // If using routing, selectTab isn't set, and routerLinkActive hasn't set any active tabs, default to the first route
```

Comment:
Is `selectedTab` meant here? I don't know what `selectTab` refers to.

Reply:
yup, that's a typo